Cork/Classes/Controller/View Controllers/CRKComposeViewController.m

```diff
@@ -98,6 +103,21 @@ - (void)send{
                 NSLog(@"%@",error);
             }
         }];
+        dispatch_async(dispatch_get_main_queue(), ^{
+            NSManagedObjectContext *mainContext = [CRKCoreDataHelper sharedHelper].persistenceController.managedObjectContext;
+            CRKUser *user = [CRKUser uniqueObjectWithIdentifier:sender.id inContext:mainContext];
+            CRKConversation *convo = [CRKConversation conversationWithUser:user inContext:mainContext];
+            
+            CRKConversationViewController *convoVC = [[CRKConversationViewController alloc] init];
+            convoVC.conversation = convo;
+            convoVC.readContext = mainContext;
+            convoVC.sender = user;
+            
+            [self.presentingViewController showViewController:convoVC sender:nil];
```

Comment:
It complete's it's animation before we dismiss. So it dismisses into the convoVC